jobs/candlepinPipelineJob.groovy

```diff
@@ -0,0 +1,40 @@
+import jobLib.rhsmLib
+String baseFolder = rhsmLib.candlepinJobFolder
+
+String githubOrg = binding.variables['CANDLEPIN_JENKINS_GITHUB_ORG'] ?: 'candlepin'
+
+pipeline_helper = job("$baseFolder/candlepin pipeline helper") {
+    logRotator {
+        numToKeep(1)
+    }
+    steps {
+        downstreamParameterized {
+            trigger('subscription-manager') {
```

Comment:
When I load this into vagrant I get a: 
```
Unresolved

subscription-manager
```

when you go to this job;  maybe this needs to be the full project name? `subscription-manager/subscription-manager`